pkg/tiles/binarytiles/binary_tile.go

```diff
@@ -25,11 +32,14 @@ type BinaryTile int64
 //  - next four meeple bits are the corners (same as with fields)
 //  - the last meeple bit is the center
 //  - the owner bits is just one-hot-encoded player ID. (ID(1) = 00...001, ID(2) = 00...010, etc.)
+//  - is placed bit is always 1 on all placed tiles, and 0 on the non-placed tiles
+//  - position bits are 8-bit reptesentations of tile position + 128, so that there are no negative numbers
```

Comment:
What is the reasoning behind converting to non-negative numbers?

Reply:
I had some problems with converting positions from int8 to uint64 (a.k.a. to BinaryTile) and writing test cases for that, but I figured it out now